src/components/snapshots/routes/index.ts

```diff
@@ -0,0 +1,22 @@
+import { FastifyPluginCallback } from 'fastify'
+import fastifyStatic from 'fastify-static'
+import { SNAPSHOT_FOLDER } from '../constants'
+import routeListSnapshots from './routeListSnapshots'
+import routeTakeSnapshot from './routeTakeSnapshot'
+import routeUseSnapshot from './routeUseSnapshot'
+import routeUploadSnapshot from './routeUploadSnapshot'
+
+const snapshotRoutes: FastifyPluginCallback<{ prefix: string }> = (server, _, done) => {
+  server.register(fastifyStatic, {
+    root: SNAPSHOT_FOLDER,
+    prefix: '/files/',
+  })
+  server.get('/list', routeListSnapshots)
+  server.post('/take', routeTakeSnapshot)
+  server.post('/use', routeUseSnapshot)
```

Comment:
Since these two just use a single query parameter, why not GET methods? Seems like "/upload" is the only one that needs to be POST?

Reply:
Although I guess it fits the convention of a POST request causing a "change" or side-effect on server, so fine as is 👍

Reply:
I was thinking that config (options) could be passed through from front end (like the filter for template etc..). Can change to get for now, would TO-DO be ok for now ?

8d8a13b03351e358d3ff5b6a1f3bd4574624b181

Reply:
Yeah, sounds good -- you might have missed my prev comment before yours, so fine to leave as-is.